starter-code/index.html

```diff
@@ -1,31 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
+<link rel="stylesheet" type="text/css" href="./css/main.css">
+<meta charset="UTF-8">
 <head>
   <title>Instagram</title>
 </head>
+
+
+
 <body>
-  Instagram
+<div class = "top">
+  <img src="./images/brand.png" alt="Instagram">
+</div>
+
+
+<div class="img1"> <img src="./images/phones.png" alt="Iphone"></div>
+<div class="boxWithText">
+  <h1>Capture and Share the World's Moments</h1>
+
+  <p>Instagram is a fast, beautiful and fun way 
+  to share your life with friends and family.</p>
+
+  <p>Take a picture or video, choose a filter to 
+  transform its look and feel, then post to Instagram &mdash; it's that easy. 
+  You can even share to Facebook, Twitter, Tumblr and more. 
+  It's a new way to see the world.</p>
 
-  Capture and Share the World's Moments
+  <p>Oh yeah, did we mention it's free?</p>
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+  <div>  
+  <img src="./images/badge-iphone.png" alt="Download on the App Store">
+  <img src="./images/badge-android.png" alt="Get it on Google Play"> 
+</div>
+</div>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
 
-  Oh yeah, did we mention it's free?
+  <footer class="footerInstagram">
```

Comment:
this needs to be centered on the page and have a little padding on the bottom to move it up a little